tasks/reduce-snapshot/reduce-snapshot.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: reduce-snapshot
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to reduce snapshot
+  params:
+    - name: SNAPSHOT
+      type: string
+    - name: SINGLE_COMPONENT
+      type: string
+    - name: SINGLE_COMPONENT_CUSTOM_RESOURCE
+      type: string
+    - name: SINGLE_COMPONENT_CUSTOM_RESOURCE_NS
+      type: string
+      default: ""
+    - name: SNAPSHOT_PATH
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  steps:
+    - name: reduce
+      env:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: SINGLE_COMPONENT
+          value: $(params.SINGLE_COMPONENT)
+        - name: CUSTOM_RESOURCE
+          value: $(params.SINGLE_COMPONENT_CUSTOM_RESOURCE)
+        - name: CUSTOM_RESOURCE_NAMESPACE
+          value: $(params.SINGLE_COMPONENT_CUSTOM_RESOURCE_NS)
+        - name: SNAPSHOT_PATH
+          value: $(params.SNAPSHOT_PATH)
+      image: quay.io/enterprise-contract/ec-cli:snapshot
```

Comment:
Is this the final tag? Before we had to ingest it from their configmap to ensure it was tested and new enough. This not being a sha gives me a little hesitation, but I haven't had the conversations with them about this

Reply:
I followed what EC does in their own task: https://github.com/enterprise-contract/ec-cli/blob/main/tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml#L202

But I can change if you prefer to use the configMap or straight digest.

cc: @lcarva

Reply:
I am fine with whatever digest or tag we are sure works and will continue to work, as a bad push to it would break production users immediately

Reply:
Probably a pinned reference is best. We try not to break things but sometimes that happens.

Reply:
modified to use pinned reference